storage/innobase/fil/fil0fil.cc

```diff
@@ -7469,14 +7469,45 @@ fil_temp_update_encryption(
 /** Default master key id for bootstrap */
 static const ulint ENCRYPTION_DEFAULT_MASTER_KEY_ID = 0;
 
+/** Rotate the tablespace key by new master key.
+@param[in]	space	tablespace object
+@return true if the re-encrypt suceeds */
+static
+bool
+fil_encryption_rotate_low(const fil_space_t* space)
```

Comment:
Pass space by const reference?

Reply:
I assume  is nit-pick :)  InnoDB code in general prefers const*. Left this one as is

Reply:
Yes, a nit pick, but it is a C legacy and InnoDB is incorrect here to carry over it to C++. I believe their new code uses const references too as it should, but not uniformly. OK to leave as is